### DISCUSSION TITLE
Functionality in VS Code extension to load and export snapshots as .json (and as image as well?)

### DISCUSSION THREAD

**Discussion by jrobertoln** [microsoft/SandDance#614] [Ideas]
I think that it would be a great addition to the VS Code extension if the snapshots could be exported to .json as well as to an image (e.g. in .png format), and if saved snapshots could be reloaded for resuming a data analysis session which was left unfinished.

**Comment by danmarshall** [comment]
Related: requests for image in #571 #600
